tests/installation/encrypted_volume_activation.pm

```diff
@@ -0,0 +1,27 @@
+use base "basetest";
+use strict;
+use testapi;
+
+sub run {
+    assert_screen 'encrypted_volume_activation_prompt';
+    if (get_var('ENCRYPT_CANCEL_EXISTING')) {
+	send_key 'alt-c', 5;
+	if (check_screen('encrypted_volume_activation_prompt')) {
+	    record_soft_failure 'bsc#989770';
+	    send_key 'alt-c';
+	}
+    } elsif (get_var('ENCRYPT_ACTIVATE_EXISTING')) {
+	send_key 'alt-p';
+	assert_screen 'encrypted_volume_password_prompt', 10;
```

Comment:
You should rely on at least the default timeout of 30 seconds, i.e. just write without explicitly stating the timeout. The assert_screen will successfully return as soon as the tag is found so no time is wasted. Slower values should only be used if you are testing a SUT behaviour itself which times out, e.g. grub boot prompt that is booting default after 8 seconds.